SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs

```diff
@@ -0,0 +1,67 @@
+using System.IO;
```

Comment:
Driveby - we should probably have copyrights at the top of these files -
`// Copyright (c) Improbable Worlds Ltd, All Rights Reserved`

Reply:
Ditto